app/ui/src/main/java/com/fsck/k9/activity/setup/AccountSetupBasics.java

```diff
@@ -369,6 +376,21 @@ private void onNext() {
     @Override
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         if (resultCode == RESULT_OK) {
+            if (requestCode == CHOOSE_ACCOUNT) {
+                final String accountName = data.getStringExtra(AccountManager.KEY_ACCOUNT_NAME);
+                new AndroidAccountOAuth2TokenStore(this.getApplicationContext()).authorizeApi(accountName, this, new OAuth2TokenProvider.OAuth2TokenProviderAuthCallback() {
```

Comment:
This should also be either injected or `authorizeApi` should be converted to static...